src/SFA.DAS.Reservations.Web/AppStart/AuthorizationService.cs

```diff
@@ -47,6 +47,22 @@ public static void AddAuthorizationService(this IServiceCollection services)
                         ProviderOrEmployerAssertion(policy);
                         policy.Requirements.Add(new HasEmployerViewerUserRoleOrIsProviderRequirement());
                     });
+                options.AddPolicy(
+                    PolicyNames.HasProviderGotViewerOrHigherRole
```

Comment:
Should the policy also say `OrIsEmployer` this would be consistent with the above role and would also be more explicit for those shared controller actions

Reply:
done